src/core/tests/logger.cpp

```diff
@@ -0,0 +1,169 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <iostream>
+#include <regex>
+#include <sstream>
+
+#include "openvino/core/log_util.hpp"
+#include "openvino/core/logging.hpp"
+#include "openvino/util/log.hpp"
+
+namespace ov::util::test {
+
+/* Capturing std::cout streambuffer doesn't work in CI job on Windows. It needs to be checked whether such test of
+ * logging to std::cout is doable other way. Though it's disabled.
+ */
+#define ENABLE_LOGGING_TO_STD_COUT_TESTS 0
```

Comment:
Use constexpr instead?

Reply:
Use as in other tests skip test configuration?
Or maybe make dedicated fixture like `DISABLED_StdCoutTest : public TestLogHelper`, maybe it works and #defined or fixture name change requires compile tests again.

Is optional change, as current is working

Reply:
I rearranged the tests a bit - macro removed.